web3/utils/datastructures.py

```diff
@@ -95,6 +99,15 @@ def __eq__(self, other):
         else:
             return False
 
+    @staticmethod
+    def serializeHexBytes(value):
```

Comment:
Rather than this being a staticmethod, I think I'd rather it be implemented as a stand-alone function.  I prefer keeping our classes as small as possible and I don't think this will be regularly used directly.  Probably belongs in `web3.utils.encoding` as something like `to_hex_if_bytes`.

Reply:
I agree it's better to have this standalone.

My instinct is that this should be internally handled as part of a `to_json()` method that is parallel to the other `to_bytes`, `to_hex`, etc. Example code in another comment.